ddp-workspace/projects/ddp-sdk/src/lib/services/authentication/auth0Adapter.service.ts

```diff
@@ -196,6 +196,7 @@ export class Auth0AdapterService implements OnDestroy {
         const auth0Params = {
             study_guid: this.configuration.studyGuid,
             mode: Auth0Mode.LoginOnly,
+            is_admin_client: true,
```

Comment:
Would you mind pointing at where this is consumed? In one of the Auth0 rules?

Reply:
In the linked PR: https://github.com/broadinstitute/ddp-study-server/pull/570.